docs/spark-standalone.md

```diff
@@ -374,6 +374,24 @@ To run an interactive Spark shell against the cluster, run the following command
 
 You can also pass an option `--total-executor-cores <numCores>` to control the number of cores that spark-shell uses on the cluster.
 
+#Spark Properties
```

Comment:
Space after #. Maybe call this "Client Properties"

Reply:
Added space and updated to call it "Client Properties".